netlify/functions/club/reviews.ts

```diff
@@ -64,4 +64,66 @@ router.put(
   },
 );
 
+router.get("/:workId/shared", async ({ clubId, params }, res) => {
+  if (!hasValue(params.workId)) {
+    return res(badRequest("No workId provided"));
+  }
+
+  const workId = params.workId;
+  const [reviews, members, workDetails] = await Promise.all([
+    ReviewRepository.getReviewsByWorkId(clubId, workId),
+    UserRepository.getMembersByClubId(clubId),
+    ListRepository.getWorkDetails(workId),
+  ]);
+
+  if (!workDetails) {
+    return res(badRequest("Work not found"));
+  }
+
+  const work = {
+    id: workDetails.id,
+    title: workDetails.title,
+    type: workDetails.type,
+    imageUrl: workDetails.image_url ?? undefined,
+    externalId: workDetails.external_id ?? undefined,
+    externalData: hasValue(workDetails.overview)
+      ? {
+          adult: workDetails.adult,
```

Comment:
Can we make this a util? overviewToExternalData or something? Just don't want to forget updating here if we update a field